src/um/winbase.rs

```diff
@@ -639,6 +639,8 @@ pub const SEM_FAILCRITICALERRORS: UINT = 0x0001;
 pub const SEM_NOGPFAULTERRORBOX: UINT = 0x0002;
 pub const SEM_NOALIGNMENTFAULTEXCEPT: UINT = 0x0004;
 pub const SEM_NOOPENFILEERRORBOX: UINT = 0x8000;
+pub const HANDLE_FLAG_INHERIT: DWORD = 0x00000001;
+pub const HANDLE_FLAG_PROTECT_FROM_CLOSE: DWORD = 0x00000002;
```

Comment:
These constants don't belong here. Rather they should go in between `SetFileShortNameW` and `LoadModule`.

Reply:
Done. I ended the prior `extern` block, inserted the new consts, and continued with a new `extern` block for `LoadModule`.